Project.toml

```diff
@@ -26,9 +23,6 @@ YAML = "ddb6d928-2868-570f-bddf-ab3f9cf99eb6"
 [compat]
 CodecZlib = "0.6, 0.7"
 DashBase = "0.1"
-DashCoreComponents = "2.0.0"
-DashHtmlComponents = "2.0.0"
-DashTable = "5.0.0"
```

Comment:
this was attempted in https://github.com/plotly/Dash.jl/pull/157, @waralexrom didn't like it at the time

Reply:
Seems reasonable. I will drop this commit.